src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../patternfly-utilities";
+
+.pf-c-empty-state {
+  --pf-c-empty-state--MaxWidth: #{pf-size-prem(400px)}; // one off to match design
+  --pf-c-empty-state--Padding: var(--pf-global--spacer--xl);
+  --pf-c-empty-state__icon--MarginBottom: var(--pf-global--spacer--lg);
+  --pf-c-empty-state__icon--FontSize: #{pf-size-prem(54px)}; // one off to match design
+  --pf-c-empty-state__icon--Color: #{$pf-color-black-400};
```

Comment:
should we create a global variable for this?

Reply:
same for this, @mceledonia said this is a one-off. We can create a variable for it though if you think that would be useful.

Reply:
ya I don't know, should we have a colour that can't be changed in global vars?

Reply:
Good question. FWIW, it would also be an issue in...

```
$ grep -ri '$pf-color' src/ | grep -i 'layout\|component'
src//patternfly/components/AboutModalBox/about-modal-box.scss:  --pf-c-about-modal-box--BackgroundColor: #{$pf-color-black-1000};
src//patternfly/components/AboutModalBox/about-modal-box.scss:  --pf-c-about-modal-box__button-close--BackgroundColor: #{$pf-color-black-1000};
src//patternfly/components/AboutModalBox/about-modal-box.scss:  --pf-c-about-modal-box__button-close--hover--BackgroundColor: rgba($pf-color-black-1000, .4);
src//patternfly/components/Nav/nav.scss:  --pf-c-nav__simple-list-link--hover--BackgroundColor: #{$pf-color-black-150};
src//patternfly/components/Nav/nav.scss:  --pf-c-nav__simple-list-link--active--BackgroundColor: #{$pf-color-black-150};
src//patternfly/components/Nav/nav.scss:  --pf-c-nav__simple-list-link--focus--BackgroundColor: #{$pf-color-black-150};
src//patternfly/components/Nav/nav.scss:  --pf-c-nav__horizontal-list-link--Color: #{$pf-color-black-300};
src//patternfly/components/Nav/nav.scss:  --pf-c-nav__simple-list-link--m-current--BackgroundColor: #{$pf-color-black-150};
```

Maybe we should. Seems like we use `$pf-color-black-150/1000` in enough places it should be a var.

There is already a var for `$pf-global--disabled-color--200: $pf-color-black-300` but `$pf-color-black-300` applies to non-current nav links, not a disabled thing, so we wouldn't want to use `--pf-global--disabled-color--200` there instead. Maybe the colors in the horizontal nav should change to another grey value we use elsewhere instead of matching a global disabled color?